modules/HomePage/components/TaskForm/TaskForm.tsx

```diff
@@ -25,25 +25,37 @@ const TaskForm: React.FC<TaskFormProps> = ({
   const [priority, setPriority] = useState<number>(2);
   const [dueDate, setDueDate] = useState<Date | null>(null);
 
-  const handleSubmit = (e: React.FormEvent) => {
+  const [addTodo] = useAddTodoMutation();
+
+  const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault();
     if (!title || !description || !priority || !dueDate) {
       alert("Please fill in all fields");
       return;
     }
+
+    const priorityMapping: { [key: number]: "HIGH" | "MEDIUM" | "LOW" } = {
+      1: "HIGH",
+      2: "MEDIUM",
+      3: "LOW",
+    };
```

Comment:
Let's put this inside a const file. `TaskForm.constants.ts` inside this folder

Reply:
Noted. Will fix it in an upcoming PR